tests/sandbox/test_jax.py

```diff
@@ -25,6 +25,32 @@ def compare_jax_and_py(
     simplify=False,
     must_be_device_array=True,
 ):
+    """Function to compare python graph output and jax compiled output for testing equality
+
+    In the tests below computational graphs are defined in Theano. These graphs are then passed to
+    this function which then compiles the graphs in both jax and python, runs the calculation
+    in both and checks if the results are the same
+
+    Parameters
+    ----------
+    fgraph: theano.gof.FunctionGraph
+        Theano function Graph object
+    inputs: iter
+        Inputs for function graph
+    assert_fn: func, opt
+        Assert function used to check for equality between python and jax. If not
+        provided uses np.testing.assert_allclose
+    simplify: Bool
+        ???
```

Comment:
Does this one simplify the graph? More specifically what does this do and when should we use it?

Reply:
That option should be removed.  

It was originally created as a means of modulating the type of optimizations used during compilation, since some of the default BLAS optimizations were introducing non-JAXable `Op`s into the graphs (the `"JAX"` mode now excludes those by default).

Now, that setting isn't being used and all of the tests are compiled without optimizations, which is needed to ensure that we actually test the correct `Op`s.  In other words, some optimizations will remove or replace the `Op`s we want to test and, since it's not always easy to tell when that will happen, we should simply avoid optimization altogether.

Reply:
Removed from the doc string and the conditional block